pki/database.go

```diff
@@ -109,6 +109,10 @@ func InsertIntoTable(db *sqlx.DB, tableName string, id, typ string, config []byt
 		return err
 	}
 
+	// FIXME: TODO:
+	// Clear possible SQL injection. However the db driver does not allow for variadic table names
+	// Should we perhaps have a set of prepared statements with predefined table names considering rather
+	// static database schema?
```

Comment:
I'm working on an external PKI so we'll be able to pull some uses of the DB out - I'd say if we need to make a provider store we should probably do it in something like LevelDB or Badger. So let's leave this in place for now, but note that we're planning to switch datastores and this problem will go away.